apps/prairielearn/src/server.js

```diff
@@ -470,6 +471,13 @@ module.exports.initExpress = function () {
   app.use(require('./middlewares/authn')); // authentication, set res.locals.authn_user
   app.use('/pl/api', require('./middlewares/authnToken')); // authn for the API, set res.locals.authn_user
 
+  // Must come after the authentication middleware, as we need to read the
+  // `authn_is_administrator` property from the response locals.
+  //
+  // This means that feature flag overrides will not be available for
+  // unauthenticated routes.
+  app.use(featuresMiddleware((req, res) => res.locals.authn_is_administrator));
```

Comment:
I wasn't entirely sure whether to use this of `is_administrator`; the latter only applies admin access if the `pl_access_as_administrator` cookie says that we should. I was thinking maybe there might be a use case for overriding feature flags when accessing something as a non-admin... but maybe there isn't?

Reply:
I don't see anything wrong with the code as it is -- it makes it very simple to understand who can override features.